hypothesis/lms
# New misc plugin

For:

- https://github.com/hypothesis/lms/issues/4699


Placeholder location for product differences until a better abstraction
is found.

The exact methods included in this first version correspond to the ones
that will be overridden by a D2L feature coming next.


## Testing

No functional changes, sanity check the changes with:

- Open https://hypothesis.instructure.com/courses/319/assignments/3308 as a student to test the aud claim change.

- Create a new assignment in blackboard (`Hypothesis (localhost, make devdata)`) to exercise the post configure hook and the is_gradable method.
